lib/node/nodes/join.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var dot = require('dot');
+dot.templateSettings.strip = false;
+
+var Node = require('../node');
+
+var INPUT_LEFT_ALIAS = '_cdb_analysis_input_left';
```

Comment:
I'm not totally happy with the naming used in this query, specially with aliases. @rochoa, what do you think?